### ISSUE TITLE
When resizing cmder window an extra path line appears.  

### ISSUE THREAD

**Issue by StratosNinja** [cmderdev/cmder#1000]

**Comment by StratosNinja** [comment]
By path line I mean the working directory line (eg. C:\Users)

**Comment by StratosNinja** [comment]
I just saw that this is already reported in #753  but though it's not fixed issue is closed. I'm running version 160612

**Comment by Stanzilla** [comment]
Type `clink`and tell me what it says

**Comment by MounirOnGithub** [comment]
Same issue for me, was about to create a new one but saw yours ! 

I've typed `clink` and had this 

`λ clink
Clink v0.4.5 [git:5d6c11] Copyright (c) 2012-2015 Martin Ridgers
http://mridgers.github.io/clink

Usage: <verb> <verb_options>

  Verbs:
  inject   Injects Clink into a process.
  autorun  Manage Clink's entry in cmd.exe's autorun.
  set      Adjust Clink's settings.
           ('<verb> --help' for more details).
`

**Comment by Stanzilla** [comment]
update https://github.com/mridgers/clink/releases/download/0.4.7/clink_0.4.7.zip

**Comment by vBm** [comment]
I have the same issue and i'm already on latest clink available.

```
λ clink
Clink v0.4.7 [git:dfc95a] Copyright (c) 2012-2016 Martin Ridgers
http://mridgers.github.io/clink
```

**Comment by alinnert** [comment]
I had the same issue. But it has been fixed after I've updated from clink 0.4.5 to 0.4.8.

**Comment by MartiUK** [comment]
Fixed in https://github.com/cmderdev/cmder/commit/393d7d19c5b56c94f819d8558fb5868aa7684e82.
